_layouts/tutorial_hands_on.html

```diff
@@ -221,12 +221,16 @@ <h3>{% icon congratulations %} Congratulations on successfully completing this t
 
         <hr>
 
+        <p>Please let us know what you thought of this tutorial below! This feedback should be about the training materials themselves only. If tools were not working or missing, please contact the administrators of the Galaxy instance you used. </p>
+
+        <p>Please note that your feedback will be anonymous, but <strong>publicly visible</strong> (on this <a href="https://github.com/galaxyproject/training-material/issues/989">GitHub issue</a>).</p>
```

Comment:
Can we make this optional, maybe?

Reply:
I have added a checkbox to the forms to make feedback confidential, in which case it will not be posted to the github issue

Reply:
Fantastic, thanks @shiltemann !